include/umf/providers/provider_os_memory.h

```diff
@@ -61,6 +67,8 @@ typedef enum umf_numa_mode_t {
 typedef struct umf_os_memory_provider_params_t {
     /// Combination of 'umf_mem_protection_flags_t' flags
     unsigned protection;
+    /// 'umf_memory_visibility_t' visibility mode
+    unsigned visibility;
```

Comment:
why unsigned and not umf_memory_visibility_t?

Reply:
Done